addon/components/o-s-s/slider.ts

```diff
@@ -72,8 +73,19 @@ export default class SliderComponent extends Component<SliderComponentArgs> {
 
   @action
   onRangeChange(event: InputEvent): void {
-    const value = (event.target as HTMLInputElement).value;
-    this.args.onChange(parseFloat(value));
+    const value = parseFloat((event.target as HTMLInputElement).value);
+    const max = this.args.inputOptions?.max ?? this.sliderOptions.max;
+    const min = this.args.inputOptions?.min ?? this.sliderOptions.min;
+
+    if (value > max) {
+      (event.target as HTMLInputElement).value = String(max);
+      this.args.onChange(max);
+    } else if (value < min) {
+      (event.target as HTMLInputElement).value = String(min);
+      this.args.onChange(min);
+    } else {
+      this.args.onChange(value);
+    }
```

Comment:
Does this not break existing usages of `min / max` in other components ?
Not sure I understand the reasoning behind these changes 🤔

Reply:
Retested a bunch where we use it on the soft & played around with it on storybook, all good

This is what I managed to cap max & min as html range inputs are a drag ( see what I did there ? ^^) you can't really match our need for a separate max value different from the max value of the range. So rather than work with the html range input I just removed the opportunity to select a value over max & under min